### DISCUSSION TITLE
Github stars

### DISCUSSION THREAD

**Discussion by rxstyq** [Shadow-Creation-Team/Protector#19] [Feature Request & Ideas]
Otan kapios vazi star na perni ena special role sto support server sas

**Comment by ComradeTurtle** [comment]
Approved:D

**Comment by tougrel** [comment]
+1 from me too! Wait for @z7rwastaken to check this ;)

**Comment by andreaskapsalhs** [comment]
Sure 🌟

**Comment by tougrel** [comment]
It's not possible to add this without a system that let's the user connect to his GitHub account via the bot. We will see what we can do after 0.5 is released 😄
